.gitignore

```diff
@@ -3,3 +3,5 @@
 /deps
 erl_crash.dump
 *.ez
+.idea/
```

Comment:
I suggest ignoring IDE generated files globally in your git config so they don’t need to be ignored in each repo.